pyocd/board/board_ids.py

```diff
@@ -216,7 +216,8 @@ def __init__(self, name, target, binary):
     "1910": BoardInfo(  "CY8CKIT-064B0S2-4343W","cy8c64xa_cm4",     "l1_cy8c6xxa.bin",      ),
     "1911": BoardInfo(  "CY8CKIT-064S0S2-4343W","cy8c64xa_cm4",     "l1_cy8c6xxa.bin",      ),
     "1912": BoardInfo(  "CYSBSYSKIT-01",        "cy8c6xxa",         "l1_cy8c6xxa.bin",      ),
-    "2201": BoardInfo(  "WIZwik_W7500",         "w7500",            "l1_w7500mbed.bin",     ),
+    "2201": BoardInfo(  "WIZwiki_W7500",         "w7500",            "l1_w7500mbed.bin",     ),
+    "2203": BoardInfo(  "WIZwiki_W7500P",        "w7500",            "l1_w7500mbed.bin",     ),
```

Comment:
Would you mind aligning the target type and test binary columns (and closing parentheses)? Sorry to be nitpicky. 😄

Reply:
Thanks!